bigchaindb/backend/schema.py

```diff
@@ -99,3 +103,21 @@ def init_database(connection=None, dbname=None):
     create_database(connection, dbname)
     create_tables(connection, dbname)
     create_indexes(connection, dbname)
+
+
+def validate_if_exists_asset_language(tx_body):
+    data = tx_body['asset'].get('data', {})
+
+    if data and 'language' in data:
```

Comment:
Maybe I shouldn't, but I always get concerned about relational operator precedence, such as whether `and` or `in` takes precedence. You could assuage that concern by putting `'language' in data` inside parentheses.